tests/support/server.tcl

```diff
@@ -444,6 +447,9 @@ proc start_server {options {code undefined}} {
             "keep_persistence" {
                 set keep_persistence $value
             }
+            "short_life" {
+                set short_life $value
```

Comment:
i'm not a fan of "short life", how about `fail_on_startup`, `terminate_on_startup`, `exit_on_startup`?

or alternatively, `wait_ready` (to set it to false)?
i.e. even if the server is expected to succeed the startup, maybe some test want to start measuring things before redis is ready...

Reply:
I've started with `exit_on_startup`, but for me, it (and the other similar. `fail_on_startup`, `terminate_on_startup`) it sounds like setting a bool that will cause the created server to be `exit_on_startup`.
Moreover, I looked for a general terminology that we can reuse in the future and not just for cases where there is a failure in startup.

I'll go with `wait_ready`.